agent/src/agent.ts

```diff
@@ -267,6 +268,11 @@ export class Agent extends MessageHandler {
             return typeof result === 'string' ? result : null
         })
 
+        this.registerRequest('convertGitCloneURLToCodebaseName', ({ cloneURL }) => {
```

Comment:
Conventionally, we use a prefix in the request name. I propose we use something like
```suggestion
        this.registerRequest('git/codebaseName', ({ cloneURL }) => {
```